context.go

```diff
@@ -55,9 +55,15 @@ type MessageEvent struct {
 	// Channel ID where the message was sent
 	Channel string
 
+	// ChannelName where the message was sent
+	ChannelName func() string
+
 	// User ID of the sender
 	User string
 
+	// UserName of the the sender
+	UserName func() string
```

Comment:
This should be a a `string` field.

Reply:
https://github.com/shomali11/slacker/pull/100#pullrequestreview-1019743388